xbmc/video/VideoInfoTag.cpp

```diff
@@ -722,12 +773,14 @@ void CVideoInfoTag::ParseNative(const TiXmlElement* movie, bool prioritise)
       while ((nodeDetail = nodeStreamDetails->IterateChildren("video", nodeDetail)))
       {
         CStreamDetailVideo *p = new CStreamDetailVideo();
-        XMLUtils::GetString(nodeDetail, "codec", p->m_strCodec);
+        XMLUtils::GetString(nodeDetail, "codec", value, true);
+        p->m_strCodec = StringUtils::Trim(value);
         XMLUtils::GetFloat(nodeDetail, "aspect", p->m_fAspect);
         XMLUtils::GetInt(nodeDetail, "width", p->m_iWidth);
         XMLUtils::GetInt(nodeDetail, "height", p->m_iHeight);
         XMLUtils::GetInt(nodeDetail, "durationinseconds", p->m_iDuration);
-        XMLUtils::GetString(nodeDetail, "stereomode", p->m_strStereoMode);
+        XMLUtils::GetString(nodeDetail, "stereomode", value, true);
+        p->m_strStereoMode = StringUtils::Trim(value);
```

Comment:
IMO in this cases it's really handy to say trim the value in `XMLUtils::GetString()`